sys/linux/usbip.txt

```diff
@@ -0,0 +1,33 @@
+# Copyright 2020 syzkaller project authors. All rights reserved.
+# Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+#include <drivers/usb/usbip/usbip_common.h>
+
+resource usbip_server_fd[fd]
+
+syz_usbip_server_init() usbip_server_fd
+
+syz_usbip_server_send(fd usbip_server_fd, buffer ptr[in, retSubmit], len bytesize[buffer])
+
+header {
```

Comment:
Let's keep structure names as close as possible to the [USB/IP headers](https://elixir.bootlin.com/linux/latest/source/drivers/usb/usbip/usbip_common.h#L141).

Here and below.